mlir/lib/Conversion/GENToLLVM/GENToLLVM.cpp

```diff
@@ -259,9 +259,33 @@ struct ConvertGENToLLVM final
       signalPassFailure();
   }
 };
+} // namespace
+
+//===----------------------------------------------------------------------===//
+// ConvertToLLVMPatternInterface implementation
+//===----------------------------------------------------------------------===//
 
+namespace {
+/// Implement the interface to convert MemRef to LLVM.
+struct GENToLLVMDialectInterface : public ConvertToLLVMPatternInterface {
+  using ConvertToLLVMPatternInterface::ConvertToLLVMPatternInterface;
+
+  /// Hook for derived dialect interface to provide conversion patterns
+  /// and mark dialect legal for the conversion target.
+  void populateConvertToLLVMConversionPatterns(
+      ConversionTarget &target, LLVMTypeConverter &typeConverter,
+      RewritePatternSet &patterns) const final {
+    GEN::populateGENToLLVMConversionPatterns(typeConverter, patterns);
+  }
+};
 } // namespace
 
+void mlir::GEN::registerConvertGENToLLVMInterface(DialectRegistry &registry) {
+  registry.addExtension(+[](MLIRContext *ctx, GEN::GENDialect *dialect) {
```

Comment:
Is the `+` intentional to convert to a function pointer or just a typo?

Reply:
It is. API's just like that.